### PR TITLE
Jcm example cleanup

### PR DESCRIPTION
**Pull request by cmcfadzean**
## Description
Some simple clean-up on the code examples for the elements and objects. This is just a small step toward more clear and consistent code examples and documentation.

Screen Shot 2021-04-22 at 2 47 16 PM

### CODE REVIEW

File: docs/app/views/examples/elements/breadcrumbs/_preview.html.erb [hunk 1]
```diff
@@ -1,9 +1,7 @@
-<%= sage_component SageCardBlock, {} do %>
-  <%= sage_component SageLabel, {
-    color: "draft",
-    value: "Default"
-  } %>
-<% end %>
+<!-- Docs Only Label -->
+<%= sage_component SageCardBlock, {} do %><%= sage_component SageLabel, {color: "draft",value: "Default"} %><% end %>
```
Comment (monkeypox8) [inline docs/app/views/examples/elements/breadcrumbs/_preview.html.erb:2]: I'm reasonably sure this was intended to be primarily a visual change, but I'd recommend keeping the previous heading structure intact for grouping. We do need to correct much of our heading hierarchy across the docs site (we skip `h2` on most of the example sections, for example), but the change here to a label would appear to remove the association between the description and example.

For example, in firefox's accessibility inspector for "Form Input", the version with labels appears as disassociated text sections due to the card structure:

|  current   |  proposed  |
|--------|--------|
|before|after|

File: docs/app/views/examples/elements/button/_preview.html.erb [hunk 2]
```diff
@@ -244,3 +226,14 @@ demo_configs = [
     raised: true,
   } %>
 <% end %>
+
+<!-- Docs Only Note -->
+<%= sage_component SageAlert, {
+  color: "info",
+  title: "Note:",
+  desc: "Regular buttons allow for a 'raised' shadow treatement to be toggled on or off.
+  This is on by default for 'rimary' button styles and can be toggled off with 'no_shadow'.
```
Comment (monkeypox8) [inline docs/app/views/examples/elements/button/_preview.html.erb:235]: Missing "P" in "Primary" here

File: docs/app/views/examples/elements/breadcrumbs/_preview.html.erb (hunk 1 above)
Comment (philschanely) [reply docs/app/views/examples/elements/breadcrumbs/_preview.html.erb:2]: I agree here and was wondering about specifying a tag for the label if we want the visual styling but the semantic tag. I hope to get a PR up early next week with a proposal for breaking apart things a bit more too per our discussion elsewhere.

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,35 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [3.22.0](https://github.com/Kajabi/sage-lib/compare/@kajabi/sage@3.21.0...@kajabi/sage@3.22.0) (2021-04-21)
```
Comment (philschanely) [inline docs/CHANGELOG.md:6]: 🤔 do we know why these changes got included? Perhaps this needs to be rebased against `develop`?
